src/azure-cli-core/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+* Minor changes
```

Comment:
Why do you use the minor changes here?

Reply:
for changes related with implementation details which end users are not interested, we use the wording of "Minor changes"